### PR TITLE
State action

### PR DESCRIPTION
**Pull request by jesseli2002**
https://trello.com/c/xLl7nl9E/91-review-state-machine-states-and-add-actual-event-functions-eg-deploydrogue-where-applicable

### CODE REVIEW

File: avionics/common/include/hardware/ignitor.h [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#pragma once
```
Comment (SilentSys) [inline avionics/common/include/hardware/ignitor.h:1]: Why? What's wrong with `#pragma once`?

File: t [hunk 2]
```diff
@@ -0,0 +1,67 @@
+[1mdiff --git a/avionics/common/include/states/landed.h b/avionics/common/include/states/landed.h[m
```
Comment (SilentSys) [inline t]: ??

**Review by SilentSys — CHANGES_REQUESTED** [review]

File: avionics/common/include/hardware/ignitor.h (hunk 1 above)
Comment (jesseli2002) [reply avionics/common/include/hardware/ignitor.h:1]: Technically it's not standard-compliant, and also it makes it possible to override the contents of the file in tests

File: t (hunk 2 above)
Comment (jesseli2002) [reply t]: yeah i've no idea what happened there, probably a few terminal commands gone wrong

**Review by akoen — APPROVED** [review]

**Review by SilentSys — APPROVED** [review]

**Pull request merged**